history/subscriber/persister.py

```diff
@@ -268,6 +268,37 @@ def on_put(req, resp):
                 'Logging level must be DEBUG, INFO, WARNING, ERROR or CRITICAL!', 'level')
 
 
+def str2_bool(v):
+    """
+     this function basically only converts a string to boolean
```

Comment:
If the value is Boolean, it just returns the value. In case the received value is not boolean, it is verified if the received value is a true value, being these possible values "yes", "true", "t", "1" ignoring case sensitive.